src/adapters/mc/OsConfigPolicy.mof

```diff
@@ -81,7 +81,7 @@ instance of OsConfigResource as $OsConfigResource5ref
     ReportedObjectName = "auditEnsureAllowUsersIsConfigured";
     ExpectedObjectValue = "PASS";
     DesiredObjectName = "remediateEnsureAllowUsersIsConfigured";
-    DesiredObjectValue = "PASS";
+    DesiredObjectValue = "*@*";

```

Comment:
What does this value mean?

Reply:
All authenticated users. This is the default value as per design,